fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/explain/PlannerGraphProperty.java

```diff
@@ -323,4 +331,12 @@ public PlannerGraph evaluateAtRef(@Nonnull ExpressionRef<? extends RelationalExp
             return Iterables.getOnlyElement(memberResults);
         }
     }
+
+    @Nonnull
+    @Override
+    @SpotBugsSuppressWarnings("NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE")
+    public PlannerGraph evaluateAtQuantifier(@Nonnull final Quantifier quantifier, @Nullable final PlannerGraph rangesOverResult) {
+        // since we do visit expression references in this property we can insist on rangesOverResult not being null
```

Comment:
```suggestion
        // since we visit the expression reference under the quantifier, and don't return null ourselves, we can
        // insist that rangesOverResult is never null
```

Reply:
done